src/tickcontext.ts

```diff
@@ -7,25 +7,72 @@
 import { Vex } from './vex';
 import { Tickable } from './tickable';
 import { Fraction } from './fraction';
+import { Note } from './note';
 
+export interface Metrics {
```

Comment:
Didn't we just create a `Metrics` for `Note` -- is this different?

Reply:
This is right. It had a minor impact in `note`.